packages/cactus-plugin-bungee/src/main/typescript/plugin-bungee.ts

```diff
@@ -0,0 +1,235 @@
+import {
+  Checks,
+  IJsObjectSignerOptions,
+  JsObjectSigner,
+  LogLevelDesc,
+  Logger,
+  LoggerProvider,
+  Secp256k1Keys,
+} from "@hyperledger/cactus-common";
+import { v4 as uuidV4 } from "uuid";
+import {
+  ICactusPlugin,
+  ICactusPluginOptions,
+  IPluginWebService,
+  IWebServiceEndpoint,
+} from "@hyperledger/cactus-core-api";
+import { State } from "./view-creation/state";
+import { ClientEndpointV1 } from "./web-services/client-endpoint";
+import OAS from "../json/openapi.json";
+import type { Express } from "express";
+import { Optional } from "typescript-optional";
+
+import { Server } from "http";
+import { Server as SecureServer } from "https";
+import {
+  CreateViewRequest,
+  CreateViewResponse,
+} from "./generated/openapi/typescript-axios";
+import { Snapshot } from "./view-creation/snapshot";
+import { View } from "./view-creation/view";
+import {
+  NetworkDetails,
+  ObtainLedgerStrategy,
+} from "./strategy/obtain-ledger-strategy";
+
+export interface IKeyPair {
+  publicKey: Uint8Array;
+  privateKey: Uint8Array;
+}
+
+export interface IPluginBUNGEEOptions extends ICactusPluginOptions {
+  instanceId: string;
+  keyPair?: IKeyPair;
+
+  logLevel?: LogLevelDesc;
+
+  disableSignalHandlers?: true;
+}
+
+export class PluginBUNGEE implements ICactusPlugin, IPluginWebService {
```

Comment:
ing.@eduv09 Please be more specific with the name of the plugin (and consequently the package and everything else). I know that the more verbose names are annoying but they are necessary. I wrote a little essay on why this is and will paste it below, please skim through it and then it will hopefully make more sense why we have to insist on more verbose names!

Some (strictly just) suggestions/ideas for a name: 
PluginBungeeSatpHermes
PluginBungeeHermes
PluginBungeeSatp

I realize that this plugin might not be stirctly tied to SATP nor the Hermes implementation, but we just need to give it some name that associates it with either the creators or the other packages that will be using it. It could even be called PluginBungeeRafael as far as I'm concerned because Rafael was the first person to tell me about BUNGEE a few years back. Joking but only half joking. For the sake of namespacing we got to put something there.
My favorite is `PluginBungeeHermes` but you pick whichever you like!

The promised essay on the why:

===

The naming convention for plugins is Plugin__$FUNCTIONALITY_GROUP$____$IMPLEMENTATION_FLAVOR$__

For example PluginKeychainVault where the $FUNCTIONALITY_GROUP$ is `Keychain` and the implementation flavor is `Vault` which allows
to be several different implementation flavors such as the in-memory keychain plugin and all the other ones that we will add
in the future (we also have AWS, GCP and Azure implementations).

Another example is the PluginLedgerConnectorBesu where the $FUNCTIONALITY_GROUP$ is `LedgerConnector` and the flavor
is `Besu`.

This hierarchical naming structure allows us to "scale" the project with several different kinds of implementations for
the same thing.

If you name something as simply PluginSomething (like PluginBUNGEE) then it becomes a squatter on that
namespace in the sense that if anyone wants to implement the same kind of functionality in a different
plugin later and they name theirs "PluginBUNGEEPetersCoolImplementation" then it will get confusing
because people looking at these classnames will rightfully assume that the two are in the same package
or that the original PluginBUNGEE is in some sort of parent relationship to the newer one, but the are
actually peers/siblings/etc and are both in a parent child relationship with the API server that loads
the plugins and the core API package that defines the interfaces between the API server and the plugins.

Reply:
I agree with Peter. PluginBungeeHermes sounds good to me. @eduv09 choose whatever you like the most